spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -359,4 +359,12 @@ public boolean reportColumnStats() {
         .defaultValue(SparkSQLProperties.REPORT_COLUMN_STATS_DEFAULT)
         .parse();
   }
+
+  public ParquetReaderType parquetReaderType() {
+    return confParser
+        .enumConf(ParquetReaderType::valueOf)
```

Comment:
Will it be a problem that we use `valueOf` that is case sensitive? We frequently add separate methods to convert from strings that handle that. Take a look at `PlanningMode`.